scripts/sample-reads-randomly.py

```diff
@@ -158,7 +162,9 @@ def main():
             output_file = open(output_filename, 'w')
 
         for record in reads[0]:
-            write_record(record, output_file)
+            write_record(record[2], output_file)
```

Comment:
not critical, but for clarity might suggest

    for (record1, record2) in reads[0]: ...